### PR TITLE
debugger initial ui

### PR DESCRIPTION
**Pull request by pahans**

### CODE REVIEW

File: modules/editor/web/js/debugger/channel.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            this.websocket = websocket;
+            var self = this;
+        };
+
+        Channel.prototype = Object.create(EventChannel.prototype);
+
+        Channel.prototype.constructor = Channel;
+
+
+        Channel.prototype.parseMessage = function (strMessage) {
```
Comment (sumuditha-viraj) [inline modules/editor/web/js/debugger/channel.js]: Lets add doc comments

File: modules/editor/web/js/debugger/module.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                    self._isActive = true;
+                }
+                event.preventDefault();
+                event.stopPropagation();
+            });
+
+            return this;
+
+        },
+        renderContent: function () {
```
Comment (sumuditha-viraj) [inline modules/editor/web/js/debugger/module.js]: Lets add doc comments

File: modules/editor/web/js/debugger/variable-tree.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    var Variable_Tree = function(args, debuggerInstance){
+        this._options = args;
+        this.debugger = debuggerInstance;
+    };
+
+    Variable_Tree.prototype = Object.create(EventChannel.prototype);
+    Variable_Tree.prototype.constructor = Variable_Tree;
+
+    Variable_Tree.prototype.render = function () {
```
Comment (sumuditha-viraj) [inline modules/editor/web/js/debugger/variable-tree.js]: Lets add doc comments

File: modules/editor/web/js/workspace/explorer.js [hunk 4]
```diff
@@ -187,10 +189,10 @@ define(['log', 'jquery', 'backbone', 'lodash', './explorer-item', './service-cli
                 })
             }
 
-            $(".sidebar-left").mCustomScrollbar({
-                theme: "minimal",
-                scrollInertia: 0
-            });
+            // explorerContainer.mCustomScrollbar({
```
Comment (sumuditha-viraj) [inline modules/editor/web/js/workspace/explorer.js]: Can we remove commented codes?

File: modules/editor/web/js/debugger/channel.js (hunk 1 above)
Comment (pahans) [inline modules/editor/web/js/debugger/channel.js]: resolved in https://github.com/ballerinalang/composer/pull/1996

File: modules/editor/web/js/debugger/module.js (hunk 2 above)
Comment (pahans) [inline modules/editor/web/js/debugger/module.js]: resolved in https://github.com/ballerinalang/composer/pull/1996

File: modules/editor/web/js/debugger/variable-tree.js (hunk 3 above)
Comment (pahans) [inline modules/editor/web/js/debugger/variable-tree.js]: resolved in https://github.com/ballerinalang/composer/pull/1996

File: modules/editor/web/js/workspace/explorer.js (hunk 4 above)
Comment (pahans) [inline modules/editor/web/js/workspace/explorer.js]: this is fixed in latest master

File: modules/editor/web/js/debugger/module.js (hunk 2 above)
Comment (pahans) [inline modules/editor/web/js/debugger/module.js]: fixed with https://github.com/ballerinalang/composer/pull/2011

**Pull request merged**